backend/utils/feature-flags.go

```diff
@@ -30,21 +31,30 @@ func (f FeatureFlags) Has(key string) bool {
 	return ok
 }
 
+// List returns a list of flags
+func (f FeatureFlags) List() []string {
+	var flags []string
+	for _, flag := range f {
+		r := flag.Key
+		if flag.Variant != "" {
+			r += "-" + flag.Variant
```

Comment:
can we use : to separate variant instead of -? since thats what we are doing in the header

Reply:
and i usually call the flags like "something-something"

Reply:
I just need to stringify it, so it doesn't really matter, but I want to be consistent, so for now its only lowercase and alphanumeric + "-".